src/iop/demosaic.c

```diff
@@ -596,7 +598,7 @@ static void xtrans_markesteijn_interpolate(float *out, const float *const in,
           else
           {
             // mirror a border pixel if beyond image edge
-            const int c = FCxtrans(row + yoff + 18, col + xoff + 18, NULL, xtrans);
+            const int c = FCxtrans(row + yoff + 24, col + xoff + 24, NULL, xtrans);
```

Comment:
@dtorop demosaic is the only place we're still doing these increments. Wouldn't it be better to just change the FCxtrans() function to just do +36 or something safely high like that and not need these magic numbers all throughout the code?